tests/test_units/test_metadata_class.py

```diff
@@ -109,16 +109,13 @@ def test_populate_meta_objectdata(regsurf, edataobj2):
     objdata = objectdata_provider_factory(regsurf, edataobj2)
 
     assert objdata.name == "VOLANTIS GP. Top"
-    assert mymeta["display"]["name"] == objdata.name
+    assert mymeta.display.name == objdata.name
     assert edataobj2.name == "TopVolantis"
 
     # surfaces shall have data.spec
-    assert mymeta["data"]
-    assert mymeta["data"]["spec"]
-    assert mymeta["data"]["spec"] == objdata.get_spec().model_dump(
-        mode="json",
-        exclude_none=True,
-    )
+    assert mymeta.data
+    assert mymeta.data.root.spec
+    assert mymeta.data.root.spec == objdata.get_spec()
```

Comment:
Some of these accesses require a `root` for being a `RootModel`. We could consider adding the relevant dunder methods to make access more obvious -- [Pydantic docs embrace this practice](https://docs.pydantic.dev/latest/concepts/models/#rootmodel-and-custom-root-types)

Reply:
Oh that would be nice! I think for one of the tests I wrote I had to go through two different root's to get access to the attribute I wanted and I did not like the look of it 😄